src/pages/landing/ContactPage.vue

```diff
@@ -1,19 +1,183 @@
 <script>
 import { defineComponent } from 'vue';
+
 import TheBreadcrumb from '@/components/TheBreadcrumb.vue';
+import BaseHeading from '@/components/base/BaseHeading.vue';
+import LocationPinIcon from '@/assets/icons/location-pin.svg';
+import EnvelopeIcon from '@/assets/icons/envelope.svg';
+import PhoneIcon from '@/assets/icons/phone.svg';
+import TwitterIcon from '@/assets/icons/twitter.svg';
+import FacebookIcon from '@/assets/icons/facebook.svg';
+import LinkedinIcon from '@/assets/icons/linkedin.svg';
+import InstagramIcon from '@/assets/icons/instagram.svg';
+import GithubIcon from '@/assets/icons/github.svg';
+import BehanceIcon from '@/assets/icons/behance.svg';
+import DribbbleIcon from '@/assets/icons/dribbble.svg';
 
 export default defineComponent({
   components: {
     TheBreadcrumb,
+    BaseHeading,
+    LocationPinIcon,
+    EnvelopeIcon,
+    PhoneIcon,
+    TwitterIcon,
+    FacebookIcon,
+    LinkedinIcon,
+    InstagramIcon,
+    GithubIcon,
+    BehanceIcon,
+    DribbbleIcon,
   },
 });
 </script>
 
 <template>
   <TheBreadcrumb />
   <section class="contact-page">
-    <h1>Contact Page</h1>
+    <BaseHeading class="contact-page__title" heading-size="large">Contact us</BaseHeading>
+    <div class="contact-page__items">
+      <div class="contact-page__map-container">
+        <img
+          src="../../assets/images/contact-map.png"
+          alt="contact-map"
+          class="contact-page__map-image"
+        />
+      </div>
+
+      <div class="contact-page__info-container">
+        <div class="contact-page__info-element">
+          <LocationPinIcon class="contact-page__info-icon" />
+          <span class="contact-page__info-detail">
+            Monterail Sp. z o.o. <br />
+            ul. Oławska 27-29 <br />
+            50-123 Wrocław <br />
+            Poland
+          </span>
+        </div>
+        <div class="contact-page__info-element">
+          <EnvelopeIcon class="contact-page__info-icon" />
+          <span class="contact-page__info-detail">hello@monterail.com</span>
+        </div>
+        <div class="contact-page__info-element">
+          <PhoneIcon class="contact-page__info-icon" />
+          <span class="contact-page__info-detail">+48 533 600 136</span>
+        </div>
+      </div>
+
+      <div class="contact-page__media">
+        <a href="https://www.monterail.com/" target="_blank"><TwitterIcon /></a>
+        <a href="https://www.monterail.com/" target="_blank"><FacebookIcon /></a>
+        <a href="https://www.monterail.com/" target="_blank"><LinkedinIcon /></a>
+        <a href="https://www.monterail.com/" target="_blank"><InstagramIcon /></a>
+        <a href="https://www.monterail.com/" target="_blank"><GithubIcon /></a>
+        <a href="https://www.monterail.com/" target="_blank"><BehanceIcon /></a>
+        <a href="https://www.monterail.com/" target="_blank"><DribbbleIcon /></a>
```

Comment:
As a suggestion it could be extracted into separated component that could be used here. New component could be named ExternalLink that could handle icons and/or text. Then you can just create some data structure f.ex. array of objects that describe each link. Additionally add `rel` attribute to link with values "noopener noreferrer". This is mostly to protect from `target="_blank"` [vulnerability ](https://cwe.mitre.org/data/definitions/1022.html) ([here also google link](https://web.dev/external-anchors-use-rel-noopener/)).

Reply:
Thank you for that, I'll get familiar with the topic and rewrite the code afterwards